src/components/books/details.js

```diff
@@ -0,0 +1,17 @@
+import CrudButtons from './crud-buttons';
+
+const BookDetails = (item) => {
+  const { book } = item;
+  return (
```

Comment:
- [ ] This is optional: You should consider using [destructuring of pros](https://www.geeksforgeeks.org/destructuring-of-props-in-reactjs/)